.auri/$gpv19h5j.md

```diff
@@ -0,0 +1,10 @@
+---
+package: "@lucia-auth/oauth" # package name
+type: "minor" # "major", "minor", "patch"
+---
+
+* Added scope to Apple oauth config
+
+* Updated Apple AuthUser according to Apple documentation https://developer.apple.com/documentation/sign_in_with_apple/request_an_authorization_to_the_sign_in_with_apple_server
```

Comment:
Should've mentioned it in the guide, but you need to create a new changeset for each change

Reply:
I hope it won't be too much of an issue if I just leave it like this, would it?